feature-configs/cn-ran-overlays/all-in-one/example/sriov/sriov-network-node-policy-netdevice-mh.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-mh-dpdk-site-1-fqdn-worker1
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  isRdma: false
+  nicSelector:
+    pfNames: ["ens1f1#0-0"]
+  nodeSelector:
+    node-role.kubernetes.io/site-1-fqdn-worker1: ""
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+  numVfs: 4
+  priority: 10
+  resourceName: mh_u_site_1_fqdn_worker1
```

Comment:
I'd rename the resource in a way that reflects the characteristics of the device exposed by this policy.
You want to say "I want a vfio device" for this pod rather than something related to it's placement.

Reply:
@josephdrichard

Reply:
Okay, I'll make the change.

Reply:
Was this left behind? @vitus133 / @josephdrichard